x-pack/plugins/security_solution/public/common/components/threat_match/logic_buttons.stories.tsx

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
Stories! ❤️

Reply:
I haven't actually test run them just yet. I should do that. I just copied what was already there as an example.